apps/examples/testcase/le_tc/network/tc_net_recvfrom.c

```diff
@@ -125,6 +126,8 @@ void *recvfrom_udpserver(void *args)
 	tc_net_recvfrom_p(SocketFD);
 	tc_net_recvfrom_sock_n();
 	tc_net_recvfrom_n(SocketFD);
+	TC_ASSERT_EQ_CLEANUP_RETURN("socket", SocketFD, -1, close(SocketFD), 0);
```

Comment:
you don't need to check the result of socket creation.
I think that just closing the socket is enough